### PR TITLE
Tag WebSockets.jl v0.2.2

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaWeb/WebSockets.jl](https://github.com/JuliaWeb/WebSockets.jl)
Release: [v0.2.2](https://github.com/JuliaWeb/WebSockets.jl/releases/tag/v0.2.2)
Travis: [Travis Build Status](https://travis-ci.org/JuliaWeb/WebSockets.jl/builds/222931948)
Diff: [vs v0.2.1](https://github.com/JuliaWeb/WebSockets.jl/compare/872977efc0d23c4aaa1e0f7be4efb87baa18cca9...2901cfbf7012dd902836016b734c247209ab09ce)
`requires` vs v0.2.1: 
```diff
--- 0.2.1/requires
+++ 0.2.2/requires
@@ -1,5 +1,5 @@
-julia 0.4
-Compat 0.7.16
+julia 0.5
+Compat 0.16.0
 HttpCommon 0.0.3
 HttpServer 0.0.4
 Codecs
```
cc: @shashi

### CODE REVIEW

**Comment by tkelman** [comment]
Need to use a new package minor version when dropping support for Julia 0.4

**Comment by MikeInnes** [comment]
Please check that:
- [ ] The package's CI is green for this tag (if applicable).
- [ ] The version bounds are up to date.

Then reply with `+ready` so I can merge.

**Comment by MikeInnes** [comment]
whoopsie

**Comment by tkelman** [comment]
was that supposed to be a dry run?

File: WebSockets/versions/0.2.2/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.5
+Compat 0.16.0
```
Comment (tkelman) [inline WebSockets/versions/0.2.2/requires]: looks like 0.9.5 would be enough, for `take!` - don't see anything newer being used

Comment (tkelman) [inline WebSockets/versions/0.2.2/requires]: was fixed on master in https://github.com/JuliaWeb/WebSockets.jl/commit/80c8414c412c9f9a205186f8e35614653a9c5e5f

**Comment by tkelman** [comment]
this can't be merged under this version number, so closing. attobot will make a new pr for an 0.3.0 if you tag that

**Comment by tkelman** [comment]
oh actually since 0.4 support was restored in https://github.com/JuliaWeb/WebSockets.jl/commit/80c8414c412c9f9a205186f8e35614653a9c5e5f, master would've been fine as 0.2.2 again - don't think it matters much

**Pull request closed without merging**